Nickvision.FlatpakGenerator/Program.cs

```diff
@@ -0,0 +1,128 @@
+﻿using CommandLine;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Text.Json;
+using System.Text.Json.Nodes;
+using System.Threading.Tasks;
+
+namespace Nickvision.FlatpakGenerator;
+
+public class Program
+{
+    public static async Task Main(string[] args) => await new Program().Run(args);
+    
+    /// <summary>
+    /// Construct Program
+    /// </summary>
+    private Program()
+    {
+    }
+
+    /// <summary>
+    /// Run the program
+    /// </summary>
+    /// <param name="args">Command-line arguments</param>
+    private async Task Run(string[] args)
```

Comment:
`RunAsync`

*It's standard practice to finish the name of an async method with `Async` that way someone using the method knows that it needs to be `await`ed or `.Wait`ed...hence why im so strict with it 😆*

Reply:
Damn, sorry, I can't get really used to it yet 😅

Reply:
A few years ago i wrote a discord bot in C# were literally every method called and written was async...so ever since then it's engraved in me 🤣